scripts/nginx/mylar.sh

```diff
@@ -0,0 +1,430 @@
+#!/bin/bash
+# Nginx Configuration for Mylar
+# Author: Public920
+
+user=$(cut -d: -f1 < /root/.master.info)
+
+isactive=$(systemctl is-active mylar)
+if [[ $isactive == "active" ]]; then
+    systemctl stop mylar
+fi
+
+if [[ ! -f /etc/nginx/apps/mylar.conf ]]; then
+    cat > /etc/nginx/apps/mylar.conf <<MYNG
+location /mylar {
+    include /etc/nginx/snippets/proxy.conf;
+    proxy_pass http://127.0.0.1:8090/mylar;
+    auth_basic "What's the password?";
+    auth_basic_user_file /etc/htpasswd.d/htpasswd.${user};
+}
+MYNG
+fi
+cat > /opt/mylar/config.ini <<MYCONF
+[General]
+oldconfig_version = None
+auto_update = False
+cache_dir = /opt/mylar/cache
+dynamic_update = 4
+refresh_cache = 7
+annuals_on = False
+syno_fix = False
+launch_browser = False
+wanted_tab_off = False
+enable_rss = False
+search_delay = 1
+grabbag_dir = None
+highcount = 0
+maintainseriesfolder = False
+destination_dir = None
+multiple_dest_dirs = None
+create_folders = True
+delete_remove_dir = False
+upcoming_snatched = True
+update_ended = False
+folder_scan_log_verbose = False
+interface = default
+correct_metadata = False
+move_files = False
+rename_files = False
+folder_format = $Series ($Year)
+file_format = $Series $Annual $Issue ($Year)
+replace_spaces = False
+replace_char = None
+zero_level = False
+zero_level_n = None
+lowercase_filenames = False
+ignore_havetotal = False
+ignore_total = False
+ignore_covers = True
+snatched_havetotal = False
+failed_download_handling = False
+failed_auto = False
+preferred_quality = 0
+use_minsize = False
+minsize = None
+use_maxsize = False
+maxsize = None
+autowant_upcoming = True
+autowant_all = False
+comic_cover_local = False
+add_to_csv = True
+skipped2wanted = False
+read2filename = False
+send2read = False
+nzb_startup_search = False
+unicode_issuenumber = False
+alternate_latest_series_covers = False
+show_icons = False
+format_booktype = False
+cleanup_cache = False
+secure_dir = None
+encrypt_passwords = False
+config_version = 10
+
+[Update]
+locmove = False
+newcom_dir = None
+fftonewcom_dir = False
+
+[Scheduler]
+rss_checkinterval = 20
+search_interval = 360
+download_scan_interval = 5
+check_github_interval = 360
+blocklist_timer = 3600
+
+[Weekly]
+alt_pull = 2
+pull_refresh = None
+weekfolder = False
+weekfolder_loc = None
+weekfolder_format = 0
+indie_pub = 75
+biggie_pub = 55
+pack_0day_watchlist_only = True
+reset_pullist_pagination = True
+
+[Interface]
+http_port = 8090
+http_host = 127.0.0.1
+http_username = None
+http_password = None
+http_root = /mylar
+enable_https = False
+https_cert = /opt/mylar/server.crt
+https_key = /opt/mylar/server.key
+https_chain = None
+https_force_on = False
+host_return = None
+authentication = 0
+login_timeout = 43800
+alphaindex = True
+
+[API]
+api_enabled = False
+api_key = None
```

Comment:
Might want to pre-fill this with some generated value so that we can then plug this into whatever else will support it later